npm/private/test/repositories_checked.bzl

```diff
@@ -16005,7 +16021,7 @@ def npm_repositories():
     )
 
     npm_import(
-        name = "npm__at_types_eslint-scope__registry.npmjs.org_at_types_eslint-scope_3.7.3",
+        name = "npm__at_types_eslint-scope__registry.npmjs.org__types_eslint-scope_3.7.3",
```

Comment:
I'd think this one should have the `_at`? 🤔

Reply:
Fixed